smartsim/_core/utils/helpers.py

```diff
@@ -198,9 +205,17 @@ def _installed(base_path: Path, backend: str) -> bool:
     backend_key = f"redisai_{backend}"
     backend_path = base_path / backend_key / f"{backend_key}.so"
     backend_so = Path(os.environ.get("RAI_PATH", backend_path)).resolve()
-    
+
     return backend_so.is_file()
 
+
+def redis_install_base(backends_path: t.Optional[str] = None) -> Path:
+    # pylint: disable-next=import-outside-toplevel
+    from ..._core.config import CONFIG
```

Comment:
Why not just move this import to the top?

Reply:
It results in a circular import